repos/system_upgrade/common/actors/preparepythonworkround/libraries/workaround.py

```diff
@@ -1,16 +1,27 @@
 import os
+import sys
 
 from leapp.libraries.common.utils import makedirs
 
 LEAPP_HOME = '/root/tmp_leapp_py3'
 
 
+def _get_python_dirname():
+    # NOTE: I thought about the static value: python2.7 for el7, python3.6 for
+    # el8; but in the end I've ratcher switched to this generic solution.
+    return 'python{}.{}'.format(sys.version_info.major, sys.version_info.minor)
+
+
+def _get_orig_leapp_path():
+    return os.path.join('/usr/lib', _get_python_dirname(), 'site-packages/leapp')
+
+
```

Comment:
```suggestion
def _get_orig_leapp_path():
    return sys.modules['leapp'].__path__[0]


```

Reply:
@vinzenz nice solution. But TBH I do not like the use of the private attribute. Not sure how much it's safe in upcoming 3.10+ python releases. I like the solution more as it's elegant, but not sure about the reliability in future. I am opened to discussion or update it if more people are ok with the use of this private attribute.

Reply:
It is good to think about potential future problems but I guess we can cover it with tests, right? Anyway, I am indifferent here but let's choose one :)

Reply:
Just consider this as a possible improvement for the future the only blocker were the PEM files being HTML files, that's it.